server/src/controllers/trip/deleteTrip.ts

```diff
@@ -1,7 +1,7 @@
 import { publicProcedure } from '../../trpc';
 import { UnableToDeleteTripError } from '../../helpers/errors';
 import * as validator from '../../middleware/validators/index';
-import { Trip } from '../../drizzle/methods/trip';
+import { Trip } from '../../drizzle/methods/Trip';
```

Comment:
Why this is Trip any errors to change this?

Reply:
The name of the methods were capitalized so I thought to change "trip" to "Trip". But I'll rename it to trip